keyboards/balloondogcaps/tr90/config.h

```diff
@@ -0,0 +1,22 @@
+/* Copyright 2021 Balloondog
```

Comment:
This file can be deleted.

Reply:
According to the VIA documentation I need that file.

Source: https://www.caniusevia.com/docs/configuring_qmk